doc/release-notes/0.31/0.31.md

```diff
@@ -26,6 +26,8 @@
 
 These release notes support the [Eclipse OpenJ9 0.31.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.31.0/plan).
 
+**Note:** Due to delays in the 0.31 release, a final build was not created. See the 0.32 release for Java 18 support.
```

Comment:
Please remove the last section in this file, about the full commit history. This link will never be populated.